design/ShaderLanguageFAQ.md

```diff
@@ -13,7 +14,11 @@ It is under development, but will have the following traits:
 *   It directly represents block-structured programming, including
     sequence, selection (if/else), and repetition (looping).
 *   It can be faithfully and simply converted to and from SPIR-V
-    “Shader” modules that only use GPU features available in WebGPU.
+    which only uses features available in WebGPU.
+*   It can be faithfully and simply converted to Metal Shading Language
```

Comment:
I'm confused here. Wasn't the "faithful" characteristics about round trips? In this case, we are talking about one way: WGSL -> Metal, so "faithful" doesn't appear to be a match.

Reply:
I agree with @kvark here.  I mean to have a higher bar for SPIR-V.

This distinction has since been discussed by the community group.  The outcome is codified in #588 (which references the discussion). 


I also have a "code complexity" and "efficiency of translation" goal for the SPIR-V translations.  
In my original PR #562 I updated it (concurrently with this PR)  to say  "with reasonable effort" for the MSL and HLSL cases.
Perhaps it's not worth bikeshedding over that right now.

At this point, I'll probably update my PR to point to  the WGSL spec README and preamble for further motivation and clarification of process and language goals.